dataactvalidator/config/sqlrules/c23_award_financial_3.sql

```diff
@@ -12,7 +18,7 @@ SELECT
 		FROM award_financial_assistance as sub_afa
```

Comment:
This should be using the temp table above also. And that way we also won't need the check for the submission ID in the where clause for this subquery, right?